changelogs/fragments/4953-listen-ports-facts-extend-output.yaml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - Add -a option to netstat and ss. This shows both listening and non-listening (for TCP this means established connections) sockets.
+  - Refactor netStatParse to be better readable. Add docstrings.
```

Comment:
```suggestion
  - listen_ports_acts - add ``-a`` option to ``netstat`` and ``ss``. This shows both listening and non-listening (for TCP this means established connections) sockets (https://github.com/ansible-collections/community.general/issues/4762, https://github.com/ansible-collections/community.general/pull/4953).
```

Reply:
Thank you. I'll wait for feedback form @gaetan-craft or @moonrail regarding introduction a variable to see `-a` output while keeping the current implementation output with less information as standard.